buttons.css

```diff
@@ -15,4 +15,17 @@
     border-color: var(--blue);
     background-color: var(--blue);
     color: var(--white);
+}
+
+.orange-button {
+    background-color: var(--primary-orange);
+    color: var(--white);
+    border: solid 1px var(--primary-orange);
+    padding: 0.3em;
+    margin: 0.2em;
+    font-size: 13px;
+    text-align: center;
+    display: block;
+    box-shadow: inset hsl(25deg 91% 75%) 0 1px 0 0;
+    border-radius: 0.1em;
```

Comment:
Let's remove this line. This way, the button will take it's `border-radius` from the `.btn` class and the styles will all be following the same "design system".